packages/kit-bg/src/dbs/local/LocalDbBase.ts

```
@@ -2749,18 +2728,101 @@
+  //     wallets: records,
+  //   };
+  // }
+
+  async getAllIndexedAccounts(): Promise<{
+    indexedAccounts: IDBIndexedAccount[];
+  }> {
+    const cacheKey = 'allDbIndexedAccounts';
+    const allIndexedAccountsInCache = this.dbAllRecordsCache.get(
+      cacheKey,
+    ) as IDBIndexedAccount[];
+    if (allIndexedAccountsInCache && allIndexedAccountsInCache.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)